sklearn/base.py

```diff
@@ -536,7 +536,8 @@ class TransformerMixin:
     """Mixin class for all transformers in scikit-learn."""
 
     def fit_transform(self, X, y=None, **fit_params):
-        """Fit to data, then transform it.
+        """
+        Fit to data, then transform it.
```

Comment:
@glemaitre I think for other PRs we may want to revert this before the PR is merged https://github.com/scikit-learn/scikit-learn/pull/15467#discussion_r341874046